content/developer/Theme/Smarty_template_overrides.adoc

```diff
@@ -0,0 +1,71 @@
+---
+title: "Smarty Template Overrides"
+weight: 5
+pre: "<b> </b>"
+---
+
+As of SuiteCRM 7.7, you can use a theme to change the output from a smarty template (.tpl) file. Which can be useful when you need to customise the look and feel of SuiteCRM without effecting existing functionality.
```

Comment:
I'm not an English native speaker, but it seems to me that `which` is a connector and shouldn't be used to start a sentence? Maybe just turn the preceding full stop into a comma?